int32 id
int32 type
int32 navgId
Pair[] pairs
geometry_msgs/Point center_point
geometry_msgs/Point[] corner_points
